C error while debugging vlang/gitly

### Describe the bug

C error occurs when trying to build gitly project while debugging.

### Reproduction Steps

The easiest way to reproduce the error:
1. clone https://github.com/Paul-Stern/gitly
2. switch to the'bug' branch
3. run v .

### Expected Behavior

I expected a successful build or, at most, a V error.

### Current Behavior
```
$ v .
================== C compilation error (from tcc): ==============
cc: /tmp/v_1000/bug.01JPWCFSZ61WQPKRSR4QEF2ZE8.tmp.c:62949: warning: function might return no value: 'main__App_user_feed'
cc: /tmp/v_1000/bug.01JPWCFSZ61WQPKRSR4QEF2ZE8.tmp.c:63942: error: 'sb_index3091' undeclared
...
cc: /tmp/v_1000/bug.01JPWCFSZ61WQPKRSR4QEF2ZE8.tmp.c:63942: error: 'sb_index3091' undeclared
(note: the original output was 26 lines long; it was truncated to its first 2 lines + the last line)
=================================================================
(You can pass `-cg`, or `-show-c-output` as well, to print all the C error messages).
builder error: 
==================
C error found. It should never happen, when compiling pure V code.
This is a V compiler bug, please report it using `v bug file.v`,
or goto https://github.com/vlang/v/issues/new/choose .
You can also use #help on Discord: https://discord.gg/vlang .
```

### Possible Solution

_No response_

### Additional Information/Context

_No response_

### V version

Current V version: V 0.4.10 b9d746f, timestamp: 2025-03-21 11:11:53 +0200

### Environment details (OS name and version, etc.)

$ v doctor
|V full version      |V 0.4.10 86bb2d72ea345a6aec2268e96fb8d8da23fd8d70.b9d746f
|:-------------------|:-------------------
|OS                  |linux, Ubuntu 24.10
|Processor           |12 cpus, 64bit, little endian, AMD Ryzen 5 5500U with Radeon Graphics
|Memory              |0.57GB/22.32GB
|                    |
|V executable        |/home/john/dev/gh/v/v
|V last modified time|2025-03-21 10:06:23
|                    |
|V home dir          |OK, value: /home/john/dev/gh/v
|VMODULES            |OK, value: /home/john/.vmodules
|VTMP                |OK, value: /tmp/v_1000
|Current working dir |OK, value: /home/john/dev/bug
|                    |
|Git version         |git version 2.45.2
|V git status        |weekly.2024.45-946-gb9d746fa
|.git/config present |true
|                    |
|cc version          |cc (Ubuntu 14.2.0-4ubuntu2) 14.2.0
|gcc version         |gcc (Ubuntu 14.2.0-4ubuntu2) 14.2.0
|clang version       |N/A
|tcc version         |tcc version 0.9.28rc 2024-07-31 HEAD@1cee0908 (x86_64 Linux)
|tcc git status      |thirdparty-linux-amd64 0134e9b9
|emcc version        |N/A
|glibc version       |ldd (Ubuntu GLIBC 2.40-1ubuntu3.1) 2.40

> [!NOTE]
> You can use the 👍 reaction to increase the issue's priority for developers.
>
> Please note that only the 👍 reaction to the issue itself counts as a vote.
> Other reactions and those to comments will not be taken into account.

--- a/vlib/v/parser/comptime.v
+++ b/vlib/v/parser/comptime.v
@@ -331,6 +331,10 @@ fn (mut p Parser) comptime_call() ast.ComptimeCall {
 	// the tmpl inherits all parent scopes. previous functionality was just to
 	// inherit the scope from which the comptime call was made and no parents.
 	// this is much simpler and allows access to globals. can be changed if needed.
+	p.open_scope()
+	defer {
+		p.close_scope()
+	}
 	mut file := parse_comptime(tmpl_path, v_code, mut p.table, p.pref, mut p.scope)
 	file.path = tmpl_path
 	return ast.ComptimeCall{

